inc/plugins/MybbStuff/MyAlerts/templates/modal_content.html

```diff
@@ -14,6 +14,8 @@
             <td class="tfoot smalltext" colspan="3">
                 <a href="{$mybb->settings['bburl']}/alerts.php">{$lang->myalerts_usercp_nav_alerts}</a>
                 <span class="float_right">
+                    <a href="{$mybb->settings['bburl']}/alerts.php?action=delete_read&amp;my_post_key={$mybb->post_code}&amp;ret_link={$myalerts_return_link}"
+                       onclick="return confirm('{$lang->myalerts_delete_read_confirm}'); return false;">{$lang->myalerts_usercp_nav_delete_read}</a> | 
```

Comment:
This language string isn't a "user cp nav" language string, we may want to change the current name of the string or add a new one.